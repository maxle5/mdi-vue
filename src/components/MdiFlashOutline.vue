
  <template>
      <svg :width="size" :height="size" :viewBox="viewBox">
          <path d="M7,2H17L13.5,9H17L10,22V14H7V2M9,4V12H12V14.66L14,11H10.24L13.76,4H9Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiFlashOutline",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
  </script>
