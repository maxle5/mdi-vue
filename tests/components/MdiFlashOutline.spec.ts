
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiFlashOutline from "../../src/components/MdiFlashOutline.vue";

test("MdiFlashOutline snapshot", () => {
  const wrapper = mount(MdiFlashOutline, {});
  expect(wrapper.html()).toMatchSnapshot();
});
