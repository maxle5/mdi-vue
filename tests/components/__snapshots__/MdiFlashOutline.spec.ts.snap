// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiFlashOutline snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M7,2H17L13.5,9H17L10,22V14H7V2M9,4V12H12V14.66L14,11H10.24L13.76,4H9Z" style="fill: currentColor;"></path>
</svg>"
`;
